docs/examples/sensors-cli.md

```diff
@@ -0,0 +1,51 @@
+# Sensors via ROS 2 Command Line
+
+This page describes how to read the the Create® 3 robot's sensors using the ROS 2 command line tools.
+
+## Overview
+Reading the robot's sensors is as simple as
+```sh
+ros2 topic echo {topic}
+```
+where {topic} is the topic of interest. If you're not sure what topics are available, you can type
+```sh
+ros2 topic list
```

Comment:
Should we reference the --no-daemon --spin-time arguments here?

Reply:
Good idea.